definitions/npm/rxjs_v5.0.x/flow_v0.34.x-/rxjs_v5.0.x.js

```diff
@@ -1619,3 +1627,39 @@ declare module "rxjs/testing/TestScheduler" {
     TestScheduler: typeof rxjs$SchedulerClass
   };
 }
+
+declare module "rxjs/testing/TestScheduler" {
```

Comment:
You declared this module twice, you'll need to remove the second instance.

Reply:
Indeed, a stupid misstep